packages/inventory/src/redux/entities.js

```diff
@@ -233,8 +240,10 @@ export default {
     [ACTION_TYPES.ALL_TAGS_PENDING]: (state) => ({ ...state, allTagsLoaded: false }),
     [ACTION_TYPES.LOAD_ENTITIES_PENDING]: entitiesPending,
     [ACTION_TYPES.LOAD_ENTITIES_FULFILLED]: entitiesLoaded,
+    [ACTION_TYPES.LOAD_ENTITIES_REJECTED]: loadingRejected,
     [ACTION_TYPES.LOAD_TAGS_PENDING]: showTagsPending,
     [ACTION_TYPES.LOAD_TAGS_FULFILLED]: showTags,
+    [ACTION_TYPES.LOAD_TAGS_REJECTED]: loadingRejected,
```

Comment:
This should be `ALL_TAGS_REJECTED`
```suggestion
    [ACTION_TYPES.ALL_TAGS_REJECTED]: loadingRejected,
```

Reply:
🤦Thank you! :D

Reply:
Shouldn't it be both?

Reply:
@rvsia not really, the `LOAD_TAGS` is for single system tags. `ALL_TAGS` are for filter tags, if this one fails the table is still present.